elements/rh-skip-link/rh-skip-link-lightdom-shim.css

```diff
@@ -0,0 +1,62 @@
+rh-skip-link:not(:defined) {
+  display: block !important;
+  position: relative !important;
+}
+
+rh-skip-link:not(:defined) > a {
+  background-color: var(--rh-color-surface-lightest, #ffffff) !important;
```

Comment:
important not needed in shim, only in shadow styles
other than that Liminal Gatorade Transfuses Memory